### ISSUE TITLE
Artboard support

### ISSUE THREAD

**Issue by Risu-Chuu** [antipalindrome/Photoshop-Export-Layers-to-Files-Fast#185]
Hello, 
I have just been recommended this script from a photoshop forum. It does a lot of what I need but there is one thing else I would like :)

I have created a PSD to automate outputting emotes in multiple sizes and maintain the layer names.

To do this I use Artboards. This way I have an over view of all the artworks without jumping into seperate PSDs.
Your script at the moent doesnt export the artboard size but rather the canvas space that the artboards as a collective use.

![image](https://user-images.githubusercontent.com/98613871/151606099-e14b17bd-7ee3-454b-8105-fbe1d74e7211.png)
![image](https://user-images.githubusercontent.com/98613871/151606145-22848f24-c228-461c-a3b0-b7f4c088c375.png)

I've attatched the image of my PSD workspace and an image of one of the folders output.
(I love that you can sort everything into folders thank you <3)

**Comment by antipalindrome** [comment]
Yes this is currently a [known gap](https://github.com/antipalindrome/Photoshop-Export-Layers-to-Files-Fast#known-gaps).  While I'd love to support this in the future, I'm not sure when this will get in, as it seems a bit more complicated to handle. Sorry!

**Comment by Risu-Chuu** [comment]
Hello! Thanks for the reply.
Sorry I’m new to GitHub 😅  I didn’t know there was a knowngap section.

On the off chance that it is any help. The script for ‘export layers to files’ does support art boards. 
Good luck
